### PR TITLE
Adding apiserver proxy to feature preview

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

Adding our new service of kubevier apiserver proxy to feature flags list
## 🎥 Demo

image

### CODE REVIEW

**Review by avivtur — COMMENTED** [review]
Great work, just 2 nits

File: src/utils/hooks/useFeatures/constants.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { DEFAULT_NAMESPACE } from '@kubevirt-utils/constants/constants';
 import { K8sVerb } from '@openshift-console/dynamic-plugin-sdk';
 
 export const INSTANCE_TYPE_ENABLED = 'instanceTypesEnabled';
+export const KUBEVIRT_APISERVER_PROXY = 'kubevirtApiserverProxy';
```
[the remark this answers is not in the document]
Comment (metalice) [reply src/utils/hooks/useFeatures/constants.ts:11]: why not? :)

File: src/views/clusteroverview/SettingsTab/ClusterTab/components/EnablePreviewFeaturesSection/EnablePreviewFeaturesSection.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+const EnablePreviewFeaturesSection: FC = () => {
+  const { t } = useKubevirtTranslation();
+
+  return (
+    <ExpandSection toggleText={t('Preview features')}>
+      <Stack hasGutter>
+        <StackItem isFilled>
+          <EnableFeatureCheckbox
+            featureName={INSTANCE_TYPE_ENABLED}
+            id="tp-instance-type"
```
[the remark this answers is not in the document]
Comment (metalice) [reply src/views/clusteroverview/SettingsTab/ClusterTab/components/EnablePreviewFeaturesSection/EnablePreviewFeaturesSection.tsx:22]: sure

**Review by avivtur — APPROVED** [review]

**Comment by apinnick** [comment]
What is this feature for? Does it have an epic?

**Pull request merged**